v2/examples/spiffe-jwt-using-proxy/client/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/go-spiffe/v2/spiffetls/tlsconfig"
+	"github.com/spiffe/go-spiffe/v2/svid/jwtsvid"
+	"github.com/spiffe/go-spiffe/v2/workloadapi"
+)
+
+const (
+	serverURL  = "https://localhost:8443"
+	socketPath = "unix:///tmp/agent.sock"
+)
+
+func main() {
+	// Set a timeout to prevent from hanging if this workload is not properly registered in SPIRE.
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	clientOptions := workloadapi.WithClientOptions(workloadapi.WithAddr(socketPath))
+
+	// Create a X509Source to fetch the trust bundle as needed to verify the X509-SVID presented by the server.
```

Comment:
```suggestion
	// Create an X509Source struct to fetch the trust bundle as needed to verify the X509-SVID presented by the server.
```
If not struct, then what it is

Reply:
I would assume it is a type...however I will add the struct keyword to be more specific.